src/types.ts

```diff
@@ -11,22 +11,34 @@ export type ClientType = {
   last_name: string;
   email: string;
   phone: string;
+  postalCode: string;
```

Comment:
camelCase vs underscore?

Reply:
Ah yes, assignedTo is also camelCase. Will normalize on underscore